examples/plane-demo/README.txt

```diff
@@ -0,0 +1,26 @@
+This Blockly demo uses Closure Templates to create a multilingual application.
```

Comment:
See https://github.com/google/blockly/issues/2625

I suppose there's no harm in copying the plane demo over (since you've already done the work).

Reply:
Ah ok. Yeah copied over already, can hide from index.md if we don't want eyes on it.